File: data/EXP01/go/2135/2135.output_2.go
func wordCount(startWords []string, targetWords []string) int {
    count := 0
    wordMap := make(map[string]bool)
    for _, word := range startWords {
        wordMap[word] = true
    }
    for _, target := range targetWords {
        if checkWord(wordMap, target) {
            count++
        }
    }
    return count
}

func checkWord(wordMap map[string]bool, target string) bool {
    if _, ok := wordMap[target]; ok {
        return true
    }
    for c := 'a'; c <= 'z'; c++ {
        newWord := target + string(c)
        if _, ok := wordMap[newWord]; ok {
            return true
        }
    }
    return false
}

File: data/EXP01/python/2135/2135.output_2.py
# @lc app=leetcode id=2135 lang=python3
from collections import Counter
from typing import List

class Solution:
    def wordCount(self, startWords: List[str], targetWords: List[str]) -> int:
        count_start = [Counter(word) for word in startWords]
        count_target = [Counter(word) for word in targetWords]
        
        result = 0
        for i in range(len(targetWords)):
            found = False
            for j in range(len(startWords)):
                if all(count_target[i][c] <= count_start[j][c] for c in count_target[i]):
                    found = True
                    break
            if found:
                result += 1
        
        return result